dotcom-rendering/src/layouts/FullPageInteractiveLayout.tsx

```diff
@@ -72,7 +72,6 @@ const Renderer = ({
 			format,
 
 			element,
-			adTargeting: undefined,
```

Comment:
Not entirely sure why this was `undefined`… we may need to review if this change has an impact after merging.